transfer_transaction.go

```
@@ -581,6 +581,9 @@ func (tx *TransferTransaction) validateNetworkOnIDs(client *Client) error {
 	var err error
 	for token, tokenTransfer := range tx.tokenTransfers {
 		err = token.ValidateChecksum(client)
+		if err != nil {
```

added this because the error was not handled